src/develop/payment.md

```diff
@@ -0,0 +1,272 @@
+
+const client = new TonClient({
+  network: {
+    endpoints: [`https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`],
+    access_key: PROJECT_SECRET,
+  },
+})
+```
+  </TabItem>
+  <TabItem value="axios" label="Axios">
+
+```typescript
+import axios from 'axios'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET = ''
+
+axios.defaults.baseURL = `https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`
+axios.defaults.headers.post['Content-Type'] = 'application/json'
+if (PROJECT_SECRET) {
+  axios.defaults.auth = {
+    username: '',
+    password: PROJECT_SECRET,
+  }
+}
+```
+  </TabItem>
+</Tabs>
+
+## Code examples
+
+Following examples demonstrate how you can use the types generated from the GraphQL schema in your project.
+
+### Get account balance
+
+<Tabs>
+  <TabItem value="sdk" label="Everscale SDK" default>
+
+```typescript
+import {BlockchainQuery} from './generated/graphql'
+
+// Specify your account's address
+const ACCOUNT_ADDRESS = ''
+
+try {
+  const query = `
+    query {
+      blockchain {
+        account(
+          address: "${ACCOUNT_ADDRESS}"
+        ) {
+          info {
+            balance(format: DEC)
+          }
+        }
+      }
+    }`
+  const {result} = await client.net.query({query})
+  const blockchain: BlockchainQuery = result.data.blockchain
+  console.log(`The account balance is ${blockchain.account.info.balance / 10**9}`)
+  client.close()
+} catch (error) {
+  console.error(error)
+}
+```
+  </TabItem>
+  <TabItem value="axios" label="Axios">
+
+```typescript
+import {BlockchainQuery} from './generated/graphql'
+
+// Specify your account's address
+const ACCOUNT_ADDRESS = ''
+
+try {
+  const query = `
+    query {
+      blockchain {
+        account(
+          address: "${ACCOUNT_ADDRESS}"
+        ) {
+          info {
+            balance(format: DEC)
+          }
+        }
+      }
+    }`
+  const {data} = await axios.post('', {query})
+  const blockchain: BlockchainQuery = data.data.blockchain
+  console.log(`The account balance is ${blockchain.account.info.balance / 10**9}`)
+} catch (error) {
+  console.error(error)
+}
+```
+  </TabItem>
+</Tabs>
+
+### Get incoming messages
+
+This example shows how to query incoming messages for a specified destination account using both the Everscale SDK and Axios. This can be particularly useful for processing incoming token transfers or other transaction-related information.
+
+Keep in mind that you will need to implement your own pagination to retrieve all messages, as the maximum limit of messages returned per query is 50. Consult [this document](https://docs.evercloud.dev/reference/graphql-api/query_language#sorting-and-limiting) for more information on how to achieve this.
+
+<Tabs>
+  <TabItem value="sdk" label="Everscale SDK" default>
+
+```typescript
+import {BlockchainMessage} from './generated/graphql'
+
+// Specify your account's address
+const ACCOUNT_ADDRESS = ''
+
+try {
+  const query = `
+    query {
```

Comment:
Need to remove this collection query from samples because now it is correct to use blockchain{account{messages}} api to retrieve messages

Reply:
Fixed.